examples/enclosure_group.rb

```diff
@@ -25,6 +25,13 @@
   action :create
 end
 
+# The set_script action is only available for C7000.
```

Comment:
You can use the api_variant property in the example if you want to make it more clear